src/typeDefs/inputs.ts

```diff
@@ -339,6 +339,8 @@ export const inputs = gql`
     lastName: String
     maritalStatus: MaritalStatus
     phone: UserPhoneInput
+    applangcode: String
+    id: ID
   }
```

Comment:
rename `input UpdateUserInput` to `input UpdateUserProfileInput`

make following changes for mutation:-
```graphql
type UpdateUserProfilePayload {
  user: User
}

type Mutation {
  updateUserProfile(input: UpdateUserProfileInput!): UpdateUserProfilePayload
}
```

Reply:
Please renamed to suit the conventions @Devesh326

Reply:
Done